### ISSUE TITLE
Unable to Install 2010 64-bit redistributable on a 64-bit VM in Azure

### ISSUE THREAD

**Issue by dvmorris** [aaronparker/Install-VisualCRedistributables#35]
# Expected Behavior

```
$VcList = Get-VcList | Get-VcRedist -Path "C:\Temp\VcRedist"
Install-VcRedist -VcList $VcList -Release "2010" -Architecture "x64","x86" -Path "C:\Temp\VcRedist" -Verbose
```

I expect this to install both libraries when executed on a Windows Server 2016 64-bit VM in Azure.

# Current Behavior

The above script installs the x86 library, but prints this when attempting to install the x64 library:

```
VERBOSE: Incompatible architecture: [x64]Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update
```

# Failure Information (for bugs)

Please help provide information about the failure if this is a bug by issuing the command using the `-Verbose` command.

```
VERBOSE: Filtering releases for platform and architecture.
VERBOSE: Incompatible architecture: [x64]Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update
VERBOSE: Install: [x86]Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update
VERBOSE: Performing the operation "Install" on target "C:\Users\tdadmin\AppData\Local\Temp\3\VcRedist\2010\x86\SP1\vcredist_x86.exe /passive /norestart'".
```

**_If it is not a bug, please remove the rest of this template._**

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Run the commands provided above on a stock Windows Server 2016 Data Center VM in Azure
1. Look for output that says that the x64 VC Redistributable library did not install because of an incompatible architecture
1. Look at the Get-Bitness function to see that this method is not returning the correct value of $false when called using `Get-Bitness -Architecture 'x86'`

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* **VcRedist Version**: Use `Get-Module -ListAvailable VcRedist`

```
ModuleType Version    Name                                ExportedCommands                                                                                                                                                                                       
---------- -------    ----                                ----------------                                                                                                                                                                                       
Script     1.4.2.85   VcRedist                            {Export-VcXml, Get-InstalledVcRedist, Get-VcList, Get-VcRedist...}                                                                                                                                     
```

* **PowerShell Version**: Use `Get-PSVersion`

```
Name                           Value                                                                                                                                                                                                                             
----                           -----                                                                                                                                                                                                                             
PSVersion                      5.1.14393.2636                                                                                                                                                                                                                    
PSEdition                      Desktop                                                                                                                                                                                                                           
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                           
BuildVersion                   10.0.14393.2636                                                                                                                                                                                                                   
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                   
WSManStackVersion              3.0                                                                                                                                                                                                                               
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                               
SerializationVersion           1.1.0.1     
```

* **Operating System**:

```Windows Server 2016 Data Center Edition (stock Azure image)

**Comment by aaronparker** [comment]
Thanks for the bug report. Fixed in commit 2c02bc21052ab085383241be21e64410698a2b8e and version 1.4.3.88.

**Comment by dvmorris** [comment]
Awesome, thanks!
